src/i18n/en/docsearch.ts

```diff
@@ -4,5 +4,9 @@ export default DocSearchDictionary({
 	button: 'Search',
 	placeholder: 'Search docs',
 	shortcutLabel: 'Press / to search',
+	resultsFooterLede: 'Looking for an Astro integration or theme? Need more help?',
+	resultsFooterIntegrations: 'Astro Integrations catalog',
+	resultsFooterThemes: 'Astro Themes catalog',
```

Comment:
```suggestion
	resultsFooterIntegrations: 'Astro Integrations directory',
	resultsFooterThemes: 'Astro Themes showcase',
```
Instead of a word with multiple spellings, perhaps?